spec/mailers/previews/claims_api_unsuccessful_report_mailer_preview.rb

```diff
@@ -70,40 +70,51 @@ def call_factories
   end
 
   def make_claims
-    FactoryBot.build(:auto_established_claim_v2, :errored)
-    FactoryBot.build(:auto_established_claim, :errored)
-
-    FactoryBot.build(:auto_established_claim_va_gov, created_at: Time.zone.now)
-    FactoryBot.build(:auto_established_claim_va_gov, created_at: Time.zone.now)
-    FactoryBot.build(:auto_established_claim_va_gov, created_at: Time.zone.now)
-    FactoryBot.build(:auto_established_claim_va_gov, created_at: Time.zone.now)
-
-    FactoryBot.build(:auto_established_claim_v2, :errored)
-    FactoryBot.build(:auto_established_claim_v2, :pending)
-    FactoryBot.build(:auto_established_claim_without_flashes_or_special_issues)
-    FactoryBot.build(:auto_established_claim_without_flashes_or_special_issues)
-    FactoryBot.build(:auto_established_claim_with_supporting_documents)
-    FactoryBot.build(:auto_established_claim)
+    # ClaimsApi::AutoEstablishedClaim.where(created_at: @from..@to).destroy_all
```

Comment:
Here and below - what are these comments for?

Reply:
If you run the preview, after the second run you may appreciate those lines.